api/tests/team/unit/application/api/models/organization-membership.test.js

```diff
@@ -0,0 +1,53 @@
+import { roles } from '../../../../../../lib/domain/models/Membership.js';
+import { OrganizationMembership } from '../../../../../../src/team/application/api/models/organization-membership.js';
+import { expect } from '../../../../../test-helper.js';
+
+describe('Team | Unit | Application | API | Model | OrganizationMembership', function () {
+  describe('#isMember', function () {
+    it('should instantiate a OrganizationMembership from a user with member role', function () {
```

Comment:
Le standard de notre équipe au niveau des tests, est de ne plus utiliser le terme _should_.

```suggestion
    it('instantiates a OrganizationMembership from a user with member role', function () {
```